README.md

```diff
@@ -19,10 +20,49 @@ via autowiring.
 
 ## Features
 
-- Invoke callable or create an object of a given class.
-- Resolve dependencies by parameter types using the given [PSR-11](http://www.php-fig.org/psr/psr-11/) container.
-- Pass concrete dependency instances by type.
-- Pass arguments by name.
```

Comment:
This fact is lost.

Reply:
Again: "Resolves dependencies by parameter name" and " Accepts additional custom arguments from an array"

Reply:
That doesn't reflect the original. It was about the ability to do something like this:

```php
function test(CacheInterface $cache, LoggerInterface $log, $value) { ... }

$injector = new Injector($container);
echo $injector->invoke('test', ['value' => 42]);
```

In this case it will explicitly set `$value` to 42 while getting the rest from container.

Reply:
Well, I tried to say exactly that. But I agree that it's still not perfect. I'll try to come up with a better summary.

Reply:
Updated.